app/src/main/java/com/infomaniak/mail/utils/FetchMessagesManager.kt

```diff
@@ -211,7 +211,7 @@ class FetchMessagesManager @Inject constructor(
         // We can leave safely.
         if (message.isSeen) return true
 
-        val formattedPreview = message.preview.ifBlank { null }?.let { "\n${it.trim()}" } ?: ""
+        val formattedPreview = message.getFormattedPreview(appContext)
         val body = if (message.body?.value.isNullOrBlank()) {
             formattedPreview
         } else {
```

Comment:
We should have had a non blank body, how come the notification did not go into the `else` case?

Reply:
I don't have enough history in the project, so I don't know if this is normal, but here the body is null.

Reply:
After investigation with @sirambd. The body is not loaded in the case of a reaction. So yes, it is normal to use the fallback to handle the case of the reaction or the encrypted message.